etl/exportergofmock/pulse_exporter_server.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/block-explorer/blob/master/LICENSE.md.
+
+package exportergofmock
+
+import (
+	"bytes"
+	"sort"
+
+	"github.com/insolar/block-explorer/etl/exporter"
+)
+
+type PulseServerMock struct {
+	*DataMock
+}
+
+func NewPulseServerMock(d *DataMock) *PulseServerMock {
+	return &PulseServerMock{d}
+}
+
+func (s *PulseServerMock) GetNextPulse(in *exporter.GetNextPulseRequest, stream exporter.PulseExporter_GetNextPulseServer) error {
+	pulses := make([]int64, 0)
+	for pKey := range s.RecordsByPulseNumber {
```

Comment:
Why `s.RecordsByPulseNumber` and not `s.Pulses`?

Reply:
done